test/openbci-sdk-test.js

```diff
@@ -248,84 +602,658 @@ describe('openbci-sdk',function() {
             });
         });
     });
-    xdescribe('#simulator', function() {
-        var running = false;
-        beforeEach(function(done) {
-            var ourBoard = new openBCIBoard.OpenBCIBoard();
-
-            ourBoard.simulatorStart().then(function() {
-                console.log('Simulator started');
-                ourBoard.on('sample',function(sample) {
-                    //OpenBCISample.debugPrettyPrint(sample);
-                });
-            }).catch(function(err) {
-                console.log('Error [simulator]: ' + err);
-            });
-            setTimeout(function() {
-                ourBoard.simulatorStop().then(function() {
-                    running = true;
-                    done();
-                },function(err) {
-                    console.log('Error: ' + err);
+
+
+});
+
+describe('#impedanceTesting', function() {
```

Comment:
I tried to do it but everything sort of broke on me and I just went with the try to make it work mentality hahah. I read that you could if you like set it up in another function or something weird like that but I couldn't figure it out.